apps/frontend/core-dapp/e2e/utils/playwright.ts

```diff
@@ -2,6 +2,7 @@
 import { Page } from '@playwright/test'
 import { ethers } from 'ethers'
 import { connectMockWallet } from './wallet-connect'
+import { TestId, TestIds } from '../../src/components/TestId'
```

Comment:
Appears to be unused

```suggestion
import {TestIds } from '../../src/components/TestId'
```